apps/web/src/components/Profile/components/PersonalInformation/PersonalInformation.tsx

```diff
@@ -87,7 +87,6 @@ const PersonalInformation = ({
         color={icon.color}
         level={pool ? "h3" : "h2"}
         size={pool ? "h4" : "h3"}
-        data-h2-font-weight="base(400)"
```

Comment:
This needs `className="font-normal"` to override the ToggleSection.Header default font-weight